### PR TITLE
Workaround the init issue of native tensorflow lib

### PR DESCRIPTION
**Pull request by jto**
Workaround https://github.com/spotify/scio/issues/1137

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -546,7 +546,33 @@ lazy val scioTensorFlow: Project = Project(
     "io.circe" %% "circe-core",
     "io.circe" %% "circe-generic",
     "io.circe" %% "circe-parser"
-  ).map(_ % circeVersion)
+  ).map(_ % circeVersion),
+  (testLoader in Test) := {
```
Comment (ravwojdyla) [inline build.sbt]: @jto awesome! Is it possible to put this in a separate file say in project dir and import - for style sake, this file is already uber complex, and this just adds fuel to the fire. What do you think?

Comment (regadas) [reply build.sbt]: 👍 on moving to separate file if possible 😄

Comment (jto) [reply build.sbt]: @ravwojdyla @regadas done :)

File: project/TensorflowTestClasspath.scala [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      ClassLoader.getSystemClassLoader().asInstanceOf[URLClassLoader]
+    val sysclass = classOf[URLClassLoader]
+    val systemAddJar = sysclass.getDeclaredMethod("addURL", classOf[URL])
+    systemAddJar.setAccessible(true)
+    tf.foreach { f =>
+      systemAddJar.invoke(sysloader, f.toURI().toURL())
+    }
+    parent
+  }
+}
```
Comment (ravwojdyla) [inline project/TensorflowTestClasspath.scala]: newline missing.

**Pull request merged**